--- Kantaya-Original/setup/SETUP.PHP	2001-11-30 22:13:24.000000000 +0700
+++ Kantaya/setup/SETUP.PHP	2021-08-17 22:01:37.191435254 +0700
@@ -38,7 +38,7 @@
      echo "</tr>\n";
      echo "<tr>\n";
      echo "<td width='30%'>Root</td>\n";
-     echo "<td width='70%'><input type='text' name='root'><td>\n";
+     echo "<td width='70%'><input type='text' name='root' value='root'><td>\n";
      echo "</tr>\n";
      echo "<tr>\n";
      echo "<td width='30%'>Password</td>\n";
@@ -46,7 +46,7 @@
      echo "</tr>\n";
      echo "<tr>\n";
      echo "<td width='30%'>Host</td>\n";
-     echo "<td width='70%'><input type='text' name='host' value='localhost'><td>\n";
+     echo "<td width='70%'><input type='text' name='host' value='db'><td>\n";
      echo "</tr>\n";
      echo "</table>\n";
      echo "<table width='100%' border=1>\n";
@@ -55,7 +55,7 @@
      echo "</tr>\n";
      echo "<tr>\n";
      echo "<td width='30%'>Nama User</td>\n";
-     echo "<td width='70%'><input type='text' name='nama_user'><td>\n";
+     echo "<td width='70%'><input type='text' name='nama_user' value='kantaya'><td>\n";
      echo "</tr>\n";
      echo "<tr>\n";
      echo "<td width='30%'>Password</td>\n";
@@ -63,7 +63,7 @@
      echo "</tr>\n";
      echo "<tr>\n";
      echo "<td width='30%'>Nama Database</td>\n";
-     echo "<td width='70%'><input type='text' name='nama_database'><td>\n";
+     echo "<td width='70%'><input type='text' name='nama_database' value='kantaya'><td>\n";
      echo "</tr>\n";
      echo "</table>\n";
      echo "<table width='100%' border=1>\n";
@@ -72,7 +72,7 @@
      echo "</tr>\n";
      echo "<tr>\n";
      echo "<td width='30%'>Path</td>\n";
-     echo "<td width='70%'><input type='text' name='basepath'><td>\n";
+     echo "<td width='70%'><input type='text' name='basepath' value='var/www/html'><td>\n";
      echo "</tr>\n";
      echo "</table>\n";
 echo "</td>\n";
