--- Kantaya-Original/buku_alamat/olah_data_kontak.php	2001-11-30 21:45:53.000000000 +0700
+++ Kantaya/buku_alamat/olah_data_kontak.php	2021-08-17 20:26:54.682650089 +0700
@@ -21,7 +21,27 @@
 <body>
 
 <?php
-error_reporting(0);
+$nama = $_POST['nama'];
+$gelar_dpn = $_POST['gelar_dpn'];
+$gelar_blk = $_POST['gelar_blk'];
+$jenis_kelamin = $_POST['jenis_kelamin'];
+$jabatan = $_POST['jabatan'];
+$kantor = $_POST['kantor'];
+$emailp = $_POST['emailp'];
+$web_personal = $_POST['web_personal'];
+$web_kantor = $_POST['web_kantor'];
+$telp_kantor = $_POST['telp_kantor'];
+$telp_hp = $_POST['telp_hp'];
+$telp_rumah = $_POST['telp_rumah'];
+$fax = $_POST['fax'];
+$alamat_kantor = $_POST['alamat_kantor'];
+$kota = $_POST['kota'];
+$kode_pos = $_POST['kode_pos'];
+$propinsi = $_POST['propinsi'];
+$negara = $_POST['negara'];
+$keterangan = $_POST['keterangan'];
+$modus = $_POST['modus'];
+
 $prmtr1 = '';
 $ktg_grup = '';
 if ($kategori == 'U') {
@@ -86,7 +106,7 @@
 						$msg = "Data Koresponden berhasil dihapus !";
 						break;
 			 }
-	 $results = mysql_query($query, $dbh) or die ("Invalid query");
+	 $results = mysql_query($query, $dbh) or die ("Invalid query:" . mysql_error());
 	 if ($results) {
 	 		echo "<script language='JavaScript'>";
 			echo "openurl('alamat.php?$prmtr','isi')</script>";
