--- Kantaya-Original/login.php	2001-11-30 21:45:52.000000000 +0700
+++ Kantaya/login.php	2021-08-17 19:47:41.435599935 +0700
@@ -14,6 +14,10 @@
  Tgl.     : 27-11-2001
  Oleh     : FB
  Revisi   : Konfigurasi File
+Revisi 3	:
+ Tgl.     : 17-08-2021
+ Oleh     : rbsoen
+ Revisi   : perbaikan sintax php
 ******************************************/
 
 //Cek Konfigurasi File
@@ -32,6 +36,10 @@
 mysql_connect ($db_host, $db_user, $db_pswd) or die ('Tidak bisa koneksi');
 mysql_select_db ($db_database) or die ('Tidak bisa koneksi ke Database kantaya');
 
+// Rev.3: globals diganti menjadi $_POST
+$username = $_POST['username'];
+$password = $_POST['password'];
+
 //Cek username dan password
 	$sql=mysql_query ("SELECT
      		kode_pengguna, 
@@ -54,7 +62,17 @@
 
 		//default adalah kantaya.css
   		if ($tampilan_css=='') $tampilan_css='kantaya';
-        session_register("kode_pengguna", "nama_pengguna", "level", "email", "unit_pengguna", "tampilan_css", "cfgfile");
+  		
+        session_start();
+  		
+        $_SESSION['kode_pengguna'] = $kode_pengguna;
+        $_SESSION['nama_pengguna'] = $nama_pengguna;
+        $_SESSION['level'] = $level;
+        $_SESSION['email'] = $email;
+        $_SESSION['unit_pengguna'] = $unit_pengguna;
+        $_SESSION['tampilan_css'] = $tampilan_css;
+        $_SESSION['cfgfile'] = $cfgfile;
+        
   		header('Location: agenda/buka_agenda.php');
   } else {
       echo "<center>";
--- Kantaya-Original/lib/koneksi_db.inc	2001-11-30 21:45:55.000000000 +0700
+++ Kantaya/lib/koneksi_db.inc	2021-08-17 19:54:49.427147976 +0700
@@ -7,21 +7,18 @@
  Oleh     : KB
  
 Revisi 1	:	
- Tgl.     :
- Oleh     :
- Revisi   :
+ Tgl.     : 17-08-2021
+ Oleh     : rbsoen
+ Revisi   : gunakan konfigurasi dalam cfg/kantaya.cfg
 
 ******************************************/
 
-$db="kantaya";
-$host="localhost";
-$username="root";
-$password="";
+require("../cfg/kantaya.cfg");
 
-if (!$dbh=mysql_connect($host, $username, $password))
+if (!$dbh=mysql_connect($db_host, $db_user, $db_pswd))
   {
     echo mysql_error();
   }
 
-mysql_select_db($db, $dbh);
+mysql_select_db($db_database, $dbh);
 ?>
--- Kantaya-Original/lib/cek_sesi.inc	2001-11-30 21:45:55.000000000 +0700
+++ Kantaya/lib/cek_sesi.inc	2021-08-17 19:44:22.324961175 +0700
@@ -7,13 +7,23 @@
  Oleh     : KB
  
 Revisi 1	:	
- Tgl.     :
- Oleh     :
- Revisi   :
+ Tgl.     : 17-08-2021
+ Oleh     : rbsoen
+ Revisi   : perbaikan sintax php
 
 ******************************************/
 
 session_start();
+
+# register variabel session
+$kode_pengguna = $_SESSION['kode_pengguna'];
+$nama_pengguna = $_SESSION['nama_pengguna'];
+$level = $_SESSION['level'];
+$email = $_SESSION['email'];
+$unit_pengguna = $_SESSION['unit_pengguna'];
+$tampilan_css = $_SESSION['tampilan_css'];
+$cfgfile = $_SESSION['cfgfile'];
+
 if (!$kode_pengguna or !$nama_pengguna or !$level) {
   header ('Location: ../index.php');
   }
