--- Kantaya-Original/agenda/olah_data_kegiatan.php	2001-11-30 21:45:53.000000000 +0700
+++ Kantaya/agenda/olah_data_kegiatan.php	2021-08-17 20:18:07.551143397 +0700
@@ -5,15 +5,26 @@
 include('../lib/fs_kalender.php');
 include('../lib/akses_unit.php');
 
+$hmb = $_POST['hmb'];
+$judul = $_POST['judul'];
+$ptgl = $_POST['ptgl'];
+$pbln = $_POST['pbln'];
+$pthn = $_POST['pthn'];
+$pjammulai = $_POST['pjammulai'];
+$pmntmulai = $_POST['pmntmulai'];
+$pjamakhir = $_POST['pjamakhir'];
+$pmntakhir = $_POST['pmntakhir'];
+$modus = $_POST['modus'];
+
 function simpan_kegiatan() {
   global $dbh, $kode_pengguna, $judul, $ptipe, $spjhari, $pthn, $pbln, $ptgl,
 				 $jamntml, $jamntsl, $kode_pengulangan, $deskripsi, $sftshar, $sharing_publik;
 				 if (!isset($ptipe) or $ptipe=='') { $ptipe = "Lainnya"; }
 				 if ($spjhari == 1) { $jamntml = "00:00:00"; $jamntsl= "23:59:59"; }
-				 $query  = "INSERT INTO agenda ";
-				 $query .= "VALUES ('', $kode_pengguna, '$judul', '$ptipe', '$pthn.$pbln.$ptgl',";
-				 $query .= "'$jamntml','$jamntsl',$kode_pengulangan,0,'$deskripsi',";
-				 $query .= "curdate(),curdate(),'$sharing_publik','$sftshar')";
+				 $query  = "INSERT INTO agenda(pemilik, judul, tipe, tgl_mulai, waktu_mulai, waktu_selesai, tgl_selesai, deskripsi, tgl_dibuat,sharing_publik,sifat_sharing) ";
+				 $query .= "VALUES ($kode_pengguna, '$judul', '$ptipe', '$pthn.$pbln.$ptgl',";
+				 $query .= "'$jamntml','$jamntsl','$pthn.$pbln.$ptgl','$deskripsi',";
+				 $query .= "curdate(),'$sharing_publik','$sftshar')";
 				 $hsl = mysql_query($query,$dbh); if(!$hsl) {echo  "@simpan kegiatan:".mysql_error();}
 				 return $hsl;
 }
